sympy/series/limits.py

```diff
@@ -136,12 +136,13 @@ def doit(self, **hints):
 
         if not e.has(z):
             return e
-        #limit is not working with binomial,gruntz raise an exception
+
```

Comment:
I didn't mean put line breaks after the `#` I meant to change `#foo` to `# foo` -- space after the `#`

a space after the `binomial,`, too